src/main/java/ColorSupplier.java

```diff
@@ -0,0 +1,10 @@
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final Random RANDOM = new Random();
+
+    public static Color getRandomColor() {
```

Comment:
Avoid the use of static methods in your solution. Static methods are generally considered bad practice. Instead of using a static method, create an instance of the class whose method you want to call.